packages/runner-ct/src/specs/spec-file.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { observer } from 'mobx-react'
+import State from '../lib/state'
+
+interface SpecFileProps {
+  state: State,
+  // eslint-disable-next-line
+  spec: import('./make-spec-hierarchy').SpecFile
```

Comment:
I have never even seen this syntax before! crazy. Can we just do a regular import, though?

Reply:
Yes we can, but it is confusing cause the file itself has a name `SpecFile`

Reply:
Right, maybe we should rename one of them. Anyway not a blocker, up to you, I will hit approve for now